src/main/webapp/scripts/script.js

```diff
@@ -134,3 +129,147 @@ function geocodeLatLng(position) {
+var speed = 20;
+var steps = 40;
+var step = 1;
+
+var red_col_1, red_col_2, red_col_1_b, red_col_2_b;
+var green_col_1, green_col_2, green_col_1_b, green_col_2_b;
+var blue_col_1, blue_col_2, blue_col_1_b, blue_col_2_b;
+
+var rgb_red_from, rgb_green_from, rgb_blue_from;
+var rgb_red_to, rgb_green_to, rgb_blue_to;
+var rgb_red_now, rgb_green_now, rgb_blue_now;
+
+var rgb_red_from_b, rgb_green_from_b, rgb_blue_from_b;
+var rgb_red_to_b, rgb_green_to_b, rgb_blue_to_b;
+var rgb_red_now_b, rgb_green_now_b, rgb_blue_now_b;
+
+var color_A = 0;
+var color_B = 1;
+var color_C = 1;
+var color_D = 2;
+
+var browser_infos = navigator.userAgent;
+var ie_4 = document.all&&!document.getElementById;
+var ie_5 = document.all&&document.getElementById&&!browser_infos.match(/Opera/);
+var ns_4 = document.layers;
+var ns_6 = document.getElementById&&!document.all;
+var Opera = browser_infos.match(/Opera/);
+var browser_ok = ie_4||ie_5||ns_4||ns_6||Opera;
+
+function translate_rgb() {
+    var hexa = body_colors[color_A];
+    var hexa_red = hexa.substring(1,3);
+    var hexa_green = hexa.substring(3,5);
+    var hexa_blue = hexa.substring(5,7);
+    rgb_red_from = parseInt("0x" + hexa_red);
+    rgb_green_from = parseInt("0x" + hexa_green);
+    rgb_blue_from = parseInt("0x" + hexa_blue);
+    rgb_red_now = rgb_red_from;
+    rgb_green_now = rgb_green_from;
+    rgb_blue_now = rgb_blue_from;
+
+    var hexa = body_colors[color_B];
+    var hexa_red = hexa.substring(1,3);
+    var hexa_green = hexa.substring(3,5);
+    var hexa_blue = hexa.substring(5,7);
+    rgb_red_to = parseInt("0x" + hexa_red);
+    rgb_green_to = parseInt("0x" + hexa_green);
+    rgb_blue_to = parseInt("0x" + hexa_blue);
+
+    var hexa = body_colors[color_C];
+    var hexa_red = hexa.substring(1,3);
+    var hexa_green = hexa.substring(3,5);
+    var hexa_blue = hexa.substring(5,7);
+    rgb_red_from_b = parseInt("0x" + hexa_red);
+    rgb_green_from_b = parseInt("0x" + hexa_green);
+    rgb_blue_from_b = parseInt("0x" + hexa_blue);
+    rgb_red_now_b = rgb_red_from_b;
+    rgb_green_now_b = rgb_green_from_b;
+    rgb_blue_now_b = rgb_blue_from_b;
+
+    var hexa = body_colors[color_D];
+    var hexa_red = hexa.substring(1,3);
+    var hexa_green = hexa.substring(3,5);
+    var hexa_blue = hexa.substring(5,7);
+    rgb_red_to_b = parseInt("0x" + hexa_red);
+    rgb_green_to_b = parseInt("0x" + hexa_green);
+    rgb_blue_to_b = parseInt("0x" + hexa_blue);
+
+    color_A++;
+    color_B++;
+    color_C++;
+    color_D++;
+
+    if (color_A>=body_colors.length) color_A = 0;
+    if (color_B>=body_colors.length) color_B = 0;
+    if (color_C>=body_colors.length) color_C = 0;
+    if (color_D>=body_colors.length) color_D = 0;
+
+    change();
+}
+
+function change() {
+    rgb_red_now = rgb_red_now - ((rgb_red_from-rgb_red_to)/speed);
+    rgb_green_now = rgb_green_now - ((rgb_green_from-rgb_green_to)/speed);
+    rgb_blue_now = rgb_blue_now - ((rgb_blue_from-rgb_blue_to)/speed);
```

Comment:
Whitespace indentation for the operator https://google.github.io/styleguide/jsguide.html#formatting-whitespace
The same thing applies to the whole function.